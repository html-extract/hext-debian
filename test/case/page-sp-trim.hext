<a @inner-html:trim:link_text />
