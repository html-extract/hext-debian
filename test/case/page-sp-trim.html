<a>    This is a text inside this anchor     </a>
