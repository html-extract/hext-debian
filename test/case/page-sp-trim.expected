{"link_text":"This is a text inside this anchor"}
